packages/components/tests/dummy/app/templates/application.hbs

```diff
@@ -20,14 +20,43 @@
     {{outlet}}
   </main>
   <footer class="dummy-footer">
-    <p class="dummy-paragraph">
-      <a
-        class="dummy-link"
-        href="https://github.com/hashicorp/design-system/tree/main/packages/components"
-        target="_blank"
-        rel="noopener noreferrer"
-      >GitHub repo
-      </a>
-    </p>
+    <div class="dummy-row-wrapper">
+      <div class="dummy-row">
+        <p class="dummy-paragraph-title">Our Accessibility Statement</p>
+        <p class="dummy-paragraph">
+          Each of our components intends to conform to WCAG 2.1 AA standards (at a minimum). If you find an issue with
+          any component, you can let us know in any of the following ways:
+        </p>
+        <ol class="dummy-list">
+          <li>
+            Post a message in the #team-design-systems channel in our internal Slack
```

Comment:
```suggestion
            Post a message in the [#team-design-systems](https://app.slack.com/client/T024UT03C/C7KTUHNUS/thread/CPB8GS9QT-1638823666.137000) channel in our internal Slack
```

non-blocking suggestion: add the link

Reply:
I added the link, but since this is a template and not a markdown file, I couldn't accept the suggested code.